ProcessLib/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -183,18 +174,65 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
     }
 
     // Reference temperature
-    const auto& reference_temperature =
+    double const reference_temperature =
         //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS__reference_temperature}
         config.getConfigParameter<double>(
             "reference_temperature", std::numeric_limits<double>::quiet_NaN());
+    DBUG("Use '%s' as reference temperature.", "reference_temperature");
+
+    //Specific gas constant
+    double const specific_gas_constant =
```

Comment:
Since you only use the ideal gas law, you can use PhysicalConstant::IdealGasConstant (in MaterialLib/PhysicalConstant.h) directly.

Reply:
As discussed, this won't gain us anything since we still need to have the specific gas constant for the calculation.